airbyte-config/config-models/src/main/resources/types/StandardSyncInput.yaml

```diff
@@ -19,6 +21,14 @@ properties:
   prefix:
     description: Prefix that will be prepended to the name of each stream when it is written to the destination.
     type: string
+  sourceId:
+    description: Actor ID for the source used in the sync - this is used to update the actor configuration when requested.
+    type: string
+    format: uuid
+  destinationId:
+    description: Actor ID for the destination used in the sync - this is used to update the actor configuration when requested.
+    type: string
+    format: uuid
```

Comment:
sync inputs now have sourceId and destinationId so we can refer to them directly when updating configs